includes/class-freemius.php

```diff
@@ -7986,9 +8215,24 @@ private function get_addon_active_parent_license() {
                 return null;
             }
 
-            $license = new FS_Plugin_License( $result->licenses[ 0 ] );
+            $parent_license = null;
 
-            return $license;
+            if ( empty( $license_key ) ) {
+                $parent_license = $result->licenses[ 0 ];

```

Comment:
@fajardoleo is the space addition surrounding the `0` in the square brackets a WP convention?